scripts/confirm-ending.js

```diff
@@ -62,14 +64,21 @@ function confirmEnding() { }
 
 
 function confirmEndingHandler() {
-  debugger;
-
   // read user input
+  const arg1 = document.getElementById('arg1').value;
+  const arg2 = document.getElementById('arg2').value;
 
+  debugger;
   // core logic
-  const result = confirmEnding(str, ending);
+  const result = confirmEnding(arg1, arg2);
 
   // display for use
+  document.getElementById('result').innerHTML = result;
 
   // log for developers
+  console.log('Is "' + arg1 + '" ending with "' + arg2 + '" : ' + result);
```

Comment:
can you add more console logline as,
for instance, ` console.log('user first input', arg1, type: ${typeof arg1}` also for the rest. this is part of assignment